.gitignore

```diff
@@ -8,3 +8,4 @@ Thumbs.db
 *.so
 coverage.*
 
+instrumentation/github.com/gocql/gocql/example/example
```

Comment:
Inadvertent change?

Reply:
`make precommit` summons that file into existence and I'm pretty sure it shouldn't be committed.

Reply:
odd. https://github.com/open-telemetry/opentelemetry-go-contrib/pull/137 doesn't seem to imply automatically building the example in that dir, wonder why it's doing that. it should write to $GOPATH/bin instead...

Reply:
Yeah, I noticed this as well. Thanks for catching it here.